client/src/Interface.ts

```diff
@@ -66,7 +67,16 @@ export enum RequestType {
   DELETE_COMMENT = 3,
 }
 
-export interface ResponsePayload {
+export enum GameStateDispatchEventType {
+  WS_RESPONSE = 1,
+  ACK_NEW_COMMENT = 2,
```

Comment:
What is this event?

Reply:
Any event generated from the server or from the client that could affect the state of the game? I wouldn't mind suggestions for a better name.